rainier-core/src/main/scala/com/stripe/rainier/core/Continuous.scala

```diff
@@ -211,3 +209,25 @@ object Uniform {
   def apply(from: Real, to: Real): Continuous =
     standard.scale(to - from).translate(from)
 }
+
+case class ContinuousMixture(components: Map[Continuous, Real])
+    extends StandardContinuous {
+  def generator: Generator[Double] =
+    Categorical(components).generator.flatMap { d =>
+      d.generator
+    }
+
+  val support = Support.union(components.keys.map { _.support })
+
+  def realLogDensity(real: Real): Real =
+    Real
+      .sum(
+        components.map {
+          case (dist, weight) =>
+            If(dist.support.isDefinedAt(real),
```

Comment:
do we need this? can't we rely on `dist.realLogDensity` doing basically the same thing?